app/views/admin/questions/_form.html.slim

```diff
@@ -15,14 +15,14 @@
   .modal-body
     = error_notification(f)
 
-    = f.translatable_inputs do |l|
+    / TODO - improve UI for translation system
+    - I18n.available_locales.each do |l|
       = f.input :"label_#{l}", as: :text,
-          input_html: { data: { translatable: 'questions.text' } },
-          placeholder: I18n.t('questions.text', locale: l),
+          input_html: {data: {translatable: 'questions.text'}},
+          placeholder: t('questions.text', locale: l),
           label: t('questions.text', locale: l)
       = f.input :"explanation_#{l}", as: :summernote,
-          input_html: { id: 'question_explanation', data: { translatable: 'questions.explanation' } },
-          placeholder: I18n.t('questions.explanation', locale: l),
```

Comment:
Should the placeholder remain?

Reply:
oh....since it's summernote, I think we can get rid of it